requirements.txt

```
@@ -1,4 +1,4 @@
-gemd==0.12.1
+gemd==0.13.0
```

```suggestion
gemd==0.13.2
```
Releasing a 0.14 would be good, but that edition wasn't published because the feature was needed for the multicake example that I shelved for unique_name...